conf/distro/include/tcmode-external-sourcery.inc

```diff
@@ -24,7 +24,8 @@ EXTERNAL_CROSS_NOPSEUDO = "gcc g++ cpp"
 baselib = "${@d.getVar('BASE_LIB:tune-' + (d.getVar('DEFAULTTUNE') or 'INVALID')) or d.getVar('BASELIB')}"
 
 # Ensure that the licensing variables are available to the toolchain.
-export SALT_LICENSE_SERVER
+export SALT_LICENSE_SERVER SALT_EXCLUDE_LICENSES SALT_INCLUDE_LICENSES
```

Comment:
getting this error here:
```
ERROR: ParseError at /home/sokol/ws1/meta-sourcery/conf/distro/include/tcmode-external-sourcery.inc:27: unparsed line: 'export SALT_LICENSE_SERVER SALT_EXCLUDE_LICENSES SALT_INCLUDE_LICENSES'
```
works fine if i have each variable on separate line:
```
export SALT_LICENSE_SERVER 
export SALT_LOGGING_DIR 
export SALT_INCLUDE_LICENSES
export SALT_EXCLUDE_LICENSES
export SALT_PKGINFO_FILE
export SALT_LICENSE_SOURCE
```

Reply:
Ah, thanks! Oops! Also a weird limitation in the file format, that needs fixing in bitbake.